lib/matplotlib/animation.py

```diff
@@ -894,17 +890,7 @@ def grab_frame(self, **savefig_kwargs):
         else:
             return super().grab_frame(**savefig_kwargs)
 
-    def _run(self):
-        # make a duck-typed subprocess stand in
-        # this is called by the MovieWriter base class, but not used here.
-        class ProcessStandin(object):
-            returncode = 0
-
-            def communicate(self):
-                return '', ''
-
-        self._proc = ProcessStandin()
-
```

Comment:
Is removing this safe? `self._run()` is still called, I guess it is deferring to a baseclass implementation?

Reply:
The parent class (FileMovieWriter) invokes _run in finish:
```
    def finish(self):
        # Call run here now that all frame grabbing is done. All temp files
        # are available to be assembled.
        self._run()
        MovieWriter.finish(self)  # Will call clean-up
```
so if we override finish(), we don't call _run() at all.
(TBH I think the inheritance chain of methods could be refactored a bit but that's not the point of this PR.)